data/styles/clear/include/Icons.mapcss

```diff
@@ -2516,6 +2516,12 @@ node|z17-[amenity=parking],
 area|z17-[amenity=parking],
 {icon-image: parking-m.svg;font-size: 11;icon-min-distance: 10}
 
+node|z16-[amenity=parking][location=underground],
+area|z16-[amenity=parking][location=underground],
+node|z16-[amenity=parking][parking=underground],
+area|z16-[amenity=parking][parking=underground],
+{icon-image:parking_underground-m.svg;}
```

Comment:
`[amenity=parking][location=underground]` is enough,

`[amenity=parking][parking=underground]` is just duplicating actually

Reply:
fixed :)